Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -145,6 +145,24 @@ ErrorOr<void> JPEGLoadingContext::decode()
                 }
             }
         }
+
+        // Photoshop writes inverted CMYK data (i.e. Photoshop's 0 should be 255). We convert this
+        // to expected values.
+        if (cinfo.saw_Adobe_marker) {
+            for (int i = 0; i < cmyk_bitmap->size().height(); ++i) {
+                auto const& line = cmyk_bitmap->scanline(i);
+
+                for (int j = 0; j < cmyk_bitmap->size().width(); ++j) {
+                    auto const& cmyk = line[j];
+                    cmyk_bitmap->scanline(i)[j] = {
```

Comment:
If you change it to `auto& line` above, you can change this line to `line[j] = {`

Reply:
Changed :)